layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -33,7 +33,13 @@ do_deploy:append:revpi-connect-s() {
 	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
         echo "dtoverlay=dwc2" >> ${DEPLOYDIR}/bootfiles/config.txt
         echo "dr_mode=host" >> ${DEPLOYDIR}/bootfiles/config.txt
+}
 
+do_deploy:append:revpi-connect-4() {
+	# Use the RevPi Connect device tree overlay
+	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
```

Comment:
May be this has to be changed to revpi-connect-4

Reply:
Does this overlay exist in the Kunbus kernel?

Reply:
@nbuchwitz can have some input here.
I think this is the file but the device that I have seems working well without it as well.
Here is the overlay file (I think) https://github.com/RevolutionPi/linux/blob/revpi-5.10/arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

Reply:
Connect 4 and other newer RevPi models do not need this line. Even more this line can produce unwanted results and therefore should stay absent.

Background: Newer modules come with a hat eeprom which contains the needed dtoverlay and other device specific informations. This hat eeprom is read by the compute modules firmware automatically upon boot.